mlflow/projects/databricks.py

```diff
@@ -198,18 +198,13 @@ def _create_databricks_run(tracking_uri, experiment_id, source_name, source_vers
     status or log metrics/params for the run.
     """
     if tracking.is_local_uri(tracking_uri):
-        # TODO: we'll actually use the Databricks deployment's tracking URI here in the future
         eprint("WARNING: MLflow tracking URI is set to a local URI (%s), so results from "
                "Databricks will not be logged permanently." % tracking_uri)
-        return None
-    else:
-        # Assume non-local tracking URIs are accessible from Databricks (won't work for e.g.
-        # localhost)
-        return tracking._create_run(experiment_id=experiment_id,
-                                    source_name=source_name,
-                                    source_version=source_version,
-                                    entry_point_name=entry_point_name,
-                                    source_type=SourceType.PROJECT)
+    return tracking._create_run(experiment_id=experiment_id,
```

Comment:
Opted to unconditionally `create_run` and let the warning suffice. In theory, an NFS-mounted local file URI may still be able to work, so we shouldn't completely discount it.

Reply:
I think users might be annoyed if this run still starts and wastes resources, but we can try it.

Reply:
Sorry, wastes resources? This is only related to tracking, i.e., whether we go ahead and create the run object now or later.